### ISSUE TITLE
Use GNU parallel for IndelRealigner

### ISSUE THREAD

**Issue by adamjorr** [adamjorr/zypy#4]
In GATKCaller use GNU parallel to speed things up

http://www.gnu.org/software/parallel/man.html#NAME

**Comment by adamjorr** [comment]
split -d -n l/${CORES} indelaligner.intervals tmp_intervals_    #split file into ${CORES} chunks without splitting lines. each chunk is called tmp_intervals_$(seq -f %02.0f 0 ${CORES})

**Comment by adamjorr** [comment]
for i in $(seq -f %02.0f 0 10); do echo tmp_intervals_${i} >> interval_files.txt; echo out_realigned_${i} >> realigned_out.txt; done

**Comment by adamjorr** [comment]
```
split -d -n l/${CORES} indelaligner.intervals tmp_intervals_
parallel java -jar ${GATK} -T IndelRealigner -R ${REFERENCEFILE} -I dedup.bam -targetIntervals tmp_intervals_{} -o tmp_realigned_{}.bam ::: $(seq -f %02.0f 0 $((CORES-1)))
```

**Comment by adamjorr** [comment]
Then need to use xargs to sort each of the new bams and then merge

**Comment by adamjorr** [comment]
```
echo tmp_realigned_*.bam  | xargs -d' ' -n 1 -i samtools sort -@ ${CORES} -m 2G -o srt_{} {}
samtools merge -@ ${CORES} -c -p realigned.bam srt_tmp_realigned_*.bam
```

**Comment by adamjorr** [comment]
fixed in 6a2325b
